src/components/common/ColonyGrid/ColonyGrid.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { ReactElement } from 'react';
+import {
+  defineMessages,
+  FormattedMessage,
+  MessageDescriptor,
+} from 'react-intl';
+
+import Heading from '~shared/Heading';
+import Link from '~shared/Link';
+
+import { CREATE_COLONY_ROUTE } from '~routes/index';
+import { WatchedColonies } from '~types/index';
+
+import ColonyGridItem from './ColonyGridItem';
+
+import styles from './ColonyGrid.css';
+
+const displayName = 'common.ColonyGrid';
```

Comment:
I'd argue this should live under `shared` _(yeah, I know it lived under `dashboard` in the Dapp)_, but then again, it will be removed rather soon-ish once we refactor the user profile, so don't really care all that much if it stays here for now.